includes/filters-actions.php

```diff
@@ -203,6 +204,12 @@ function woolab_icdic_checkout_field_process() {
 		}
 
 	}
+	// DIC is mandatory in Slovakia, this is not a VAT number
+	else {
+		if( empty( $_POST['billing_dic'] ) && $country == 'SK' ) {
+			wc_add_notice( __( 'Enter a valid VAT number', 'woolab-ic-dic' ), 'error' );

```

Comment:
Bolo by vhodné pridať do prekladu "Enter a valid Tax ID number" keďže je to špecificky Tax ID, nie VAT reg. number

Reply:
Ano, prosím